src/many-ledger/src/storage.rs

```diff
@@ -90,41 +104,66 @@ impl LedgerStorage {
         &self.migrations
     }
 
+    fn subresource_db_key(migration_is_active: bool) -> Vec<u8> {
```

Comment:
Can you make this a function local to the file but outside the LedgerStorage type?

Reply:
This is getting nuked with the `account_identity` refactor.